src/app/shared/object-collection/shared/object-collection-element/abstract-listable-element.component.ts

```diff
@@ -24,6 +24,8 @@ export class AbstractListableElementComponent<T extends ListableObject> {
    */
   @Input() listID: string;
 
+  @Input() value: string;
```

Comment:
could you please add a typescript comment?

Reply:
Added typedocs